lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status: s, response }, header } = this;
```

Comment:
I'm :-1: on this type of change (also in get host and hostname). In my opinion it only decreases readability.